tests/PSinLinuxCloudShellImage.Tests.ps1

```diff
@@ -18,9 +18,8 @@ Describe "Various programs installed with expected versions" {
 
     It "Static Versions" {
         # These programs are installed explicitly with specific versions
-        $script:pmap["Node.JS"].Version | Should -Be '16.16.0'
+        $script:pmap["Node.JS"].Version | Should -Be '16.17.1'
```

Comment:
Whenever there is an update to NodeJS we have to update this place. Considered any possibilities to automate this? Perhaps in future?

Reply:
Yes, in the future, we will certainly consider finding ways to automate this.